ynr/apps/popolo/migrations/0052_fix_territory.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+
+
+def fix_territory(apps, schema_editor):
+    Post = apps.get_model("popolo", "Post")
+    Post.objects.exclude(
+        territory_code__in=["ENG", "WLS", "NIR", "SCT"]
+    ).update(territory_code="ENG")
```

Comment:
For YNR, I had a look over the production data and every record with an invalid value for territory is in England, so we can just set them all to England. Interestingly, the records with a bad territory code on them in WCIVF are totally different to YNR.